### PR TITLE
Bug 1706024 - Fix path to ping entry point in package.json

### PR DESCRIPTION
**Pull request by brizental**
Not sure how we didn't pick this up before, but ok.

### CODE REVIEW

**Comment by brizental** [comment]
Argh, how did this fail CI? ~Works on **my** computer~

**Review by Dexterp37 — APPROVED** [review]

**Comment by Dexterp37** [comment]
> 
> 
> Argh, how did this fail CI? ~Works on **my** computer~

The specific fix seems reasonable. It might be a different problem. Holding back r+.

**Review by Dexterp37 — APPROVED** [review]
As discussed on Element the Ci failure seem to be unrelated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 [Full changelog](https://github.com/mozilla/glean.js/compare/v0.9.0...main)
 
+* [#219](https://github.com/mozilla/glean.js/pull/219): BUGFIX: Fix path to ping entry point in package.json.
```
Comment (Dexterp37) [inline CHANGELOG.md:5]:
```suggestion
* [#219](https://github.com/mozilla/glean.js/pull/219): BUGFIX: Fix path to ping entry point in package.json.

```

**Comment by brizental** [comment]
Merging in despite of the CI failure as it is unrelated and the fix in this PR is a real bug. CI failure is apparently related to `--prefix` changing the behaviour of `link`. Will open a different PR with a fix once I figure it out.

**Pull request merged**